Sources/RouterServiceInterface/Feature.swift

```diff
@@ -1,11 +1,22 @@
 import UIKit
 
 public protocol Feature {
-    func build(
-        fromRoute route: Route?
-    ) -> UIViewController
-
+    func build(fromRoute route: Route?) -> UIViewController
     func resolve(withStore store: StoreInterface)
+    
+    /// Provides if feature is enabled to be presented, should be used when feature is controlled by toggle (feature flag, user defaults, etc)
```

Comment:
```suggestion
    /// Returns whether or not a feature can be presented. 
    /// Implement this method if the availability of your feature should be controlled. (feature flag, user defaults, iOS version, etc)
```

Improvements to English, and I removed `toggle` as a reason because you can use it for other things like iOS versions. What do you think?